src/api/roles/shared/account.py

```
@@ -0,0 +1,58 @@
+    bucket = "profile_picture"
+    filename = f"{acc.id}_{file.filename}"
+    upload_url = f"{SUPABASE_URL.rstrip('/')}/storage/v1/object/{bucket}/{filename}"
+
+    headers = {
+        "Authorization": f"Bearer {SUPABASE_SERVICE_KEY}",
+        "apikey": SUPABASE_SERVICE_KEY,
+    }
+
+    try:
+        # stream upload
+        resp = requests.put(upload_url, data=file.file, headers=headers)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)